### DISCUSSION TITLE
lazer is way laggier than stable for me. Is there going to be a high performance mode or something similar?

### DISCUSSION THREAD

**Discussion by kajxqeirscl** [ppy/osu#14188] [Q&A] [answered]
The fps drops a lot when there are many objects on the screen. The fps counter says 100 fps when it is like 40. The menu is laggy because those triangle animations I think. Sometimes the game doesnt lag much sometimes it becomes borderline unplayable.

CPU: AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx     2.10 GHz
GPU: AMD Radeon(TM) Vega 8 Graphics
RAM: 8 GB
OS: Windows 10 Home

**Comment by katelyynn** [comment]
i mean the game's still in a work in progress state, so i doubt it will stay that way. could also mention ur specs incase it could be useful for the devs etc.

**Reply by kajxqeirscl** [reply]
ok I will one sec

**Comment by smoogipoo** [comment, marked as answer]
Please check back in the future - performance is a constant work in progress.

**Comment by Tyfaspo** [comment]
I have the same specs, but my fps is around 220, it's still bad for osu! though

**Reply by peppy** [reply]
This application is multi threaded and "220fps" means less unless you mention which thread, and why you think this is an issue.
